src/Quiz.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import IntervalExercise  from './IntervalExercise.js'
+import PropTypes from "prop-types";
+
+function ContinueButton({ onClick }) {
+    return (
+        <button onClick={onClick}> Continue </button>
+    )
+}
+
+class Quiz extends Component {
+    render() {
+        var maybeContinueButton;
+        if (// we've answered correctly
+            this.props.submittedAnswers.includes(this.props.interval.toString())
+            // and this is not the last exercise
+            && this.props.currentExercise < this.props.numExercises - 1
+          ) {
+          maybeContinueButton = <ContinueButton
+            onClick={this.props.onContinueClick}
+          />
+        }
+
+        return (<div>
+            <IntervalExercise
+              interval={this.props.interval}
+              possibleAnswers={this.props.possibleAnswers}
+              submittedAnswers={this.props.submittedAnswers}
+              onAnswerClick={this.props.onAnswerClick}
+            />
+            {maybeContinueButton}
+        </div>)
```

Comment:
Indentation note: this honestly looks fine to my eye if you like it, but I would normally indent this more 
```suggestion
        return (
            <div>
                <IntervalExercise
                  interval={this.props.interval}
                  possibleAnswers={this.props.possibleAnswers}
                  submittedAnswers={this.props.submittedAnswers}
                  onAnswerClick={this.props.onAnswerClick}
                />
                {maybeContinueButton}
            </div>
        )
```